dashboard/src/components/MigrateForm/MigrateForm.tsx

```diff
@@ -0,0 +1,283 @@
+  tillerReleaseName: string;
+  chartValues: string | null | undefined;
+  chartName: string;
+  chartRepoAuth: {};
+  chartRepoName: string;
+  chartRepoURL: string;
+  repos: IAppRepository[];
+}
+
+interface IMigrationtFormState {
+  isDeploying: boolean;
+  tillerReleaseName: string;
+  chartValues: string;
+  chartVersion: string;
+  namespace: string;
+  chartName: string;
+  chartRepoName: string;
+  chartRepoURL: string;
+  chartRepoAuth: {};
+  repos: IAppRepository[];
+}
+
+class MigrateForm extends React.Component<IMigrationFormProps, IMigrationtFormState> {
+  public state: IMigrationtFormState = {
+    chartName: this.props.chartName,
+    chartRepoAuth: {},
+    chartRepoName: this.props.chartRepoName,
+    chartRepoURL: "",
+    chartValues: this.props.chartValues || "",
+    chartVersion: this.props.chartVersion,
+    isDeploying: false,
+    namespace: this.props.namespace,
+    repos: this.props.repos,
+    tillerReleaseName: this.props.tillerReleaseName,
+  };
+
+  public render() {
+    return (
+      <div>
+        <form className="container padding-b-bigger" onSubmit={this.handleDeploy}>
+          <div className="row">
+            <div className="col-8">{this.props.error && this.renderError()}</div>
+            <div className="col-12">
+              <h2>{this.props.chartID}</h2>
+            </div>
+            <div className="col-12">
+              <p>
+                In order to be able to manage {this.state.tillerReleaseName} select the repository
+                it can be retrieved from.
+              </p>
+            </div>
+            <div className="col-8">
+              <div>
+                <label htmlFor="chartRepoName">Chart Repository Name *</label>
+                <select
+                  id="chartRepoName"
+                  onChange={this.handleChartRepoNameChange}
+                  value={this.state.chartRepoName}
+                  required={true}
+                >
+                  {this.state.chartRepoName === "" && <option key="" value="" />}
+                  {this.props.repos.map(r => (
+                    <option key={r.metadata.name} value={r.metadata.name}>
+                      {r.metadata.name}
+                    </option>
+                  ))}
+                </select>
+              </div>
+              <div>
+                <label htmlFor="chartRepoURL">Chart Repository URL</label>
+                <input
+                  id="chartRepoURL"
+                  value={
+                    this.state.chartRepoURL === ""
+                      ? "(Select a repository name)"
+                      : this.state.chartRepoURL
+                  }
+                  required={true}
+                  disabled={true}
+                />
+              </div>
+              <div>
+                <p>
+                  {" "}
+                  * If the repository containing this chart is not in the list add it{" "}
+                  <a href="/config/repos"> here </a>{" "}
```

Comment:
add a full stop after.

We may want to make this an actual alert, or make this nicer with CSS

Reply:
it's quite visible right now but let me know if you have a specific suggestion